### ISSUE TITLE
null is not an object (evaluating 'ref.setNativeProps')

### ISSUE THREAD

**Issue by PavelMartynovN** [Slowyn/react-native-underline-tabbar#12]
null is not an object (evaluating 'ref.setNativeProps')

error from this block 
applyTransformToUnderline
TabBar.js:139:8
```
applyTransformToUnderline(ref: any, scaleXFactor: number, dx: number) {
    const matrix = createTranslateXScaleX(scaleXFactor, dx);
    transformOrigin(matrix, { x: -0.5, y: 0, z: 0 });
    ref.setNativeProps({
      style: {
        transform: [
          {
            matrix,
          },
        ],
      },
    });
  }
```
I use "react-native": "0.46.4"

**Comment by Slowyn** [comment]
Provide please more information. E.g. show how you use component.

**Comment by PavelMartynovN** [comment]
I'm using your example, from here https://github.com/Slowyn/react-native-underline-tabbar
and I get this error on android

```
import React, { Component } from 'react';
import {
  AppRegistry,
  StyleSheet,
  Text,
  View
} from 'react-native';
import ScrollableTabView from 'react-native-scrollable-tab-view';
import TabBar from "react-native-underline-tabbar";

const Page = ({ label }) => (
  <View style={{ flex: 1 }}>
    <Text>
      {label}
    </Text>
  </View>
);

export default class WelcomeScreen extends Component {
  render() {
    return (
      <ScrollableTabView
        style={{ paddingTop: 20 }}
        tabBarActiveTextColor="#53ac49"
        renderTabBar={() => <TabBar underlineColor="#53ac49" />}>
        <Page tabLabel={{ label: "Page #1" }} label="Page #1" />
        <Page tabLabel={{ label: "Page #2 aka Long!", badge: 3 }} label="Page #2 aka Long!" />
        <Page tabLabel={{ label: "Page #3" }} label="Page #3" />
        <Page tabLabel={{ label: "Page #4 aka Page" }} label="Page #4 aka Page" />
        <Page tabLabel={{ label: "Page #5" }} label="Page #5" />
      </ScrollableTabView>
    );
  }
}
```

**Comment by Slowyn** [comment]
Resolved with commit 2fa675eb16da2f67043af7f96c8d0bb9f829efc1
Thanks for reporting!
